.github/dependabot.yml

```diff
@@ -1,5 +1,83 @@
 version: 2
 updates:
+  - package-ecosystem: npm
+    directory: "/web/packages/design"
+    schedule:
+      interval: monthly
+      day: "sunday"
+      time: "09:00" # 9am UTC
+    open-pull-requests-limit: 10
```

Comment:
You might want to bump this if you think we are getting a storm of deps. Same for others.

Reply:
I went back and forth between "we're getting a lot so open the flood gates" and "we're getting a lot so let's try to limit it". I'm happy either way. Let's see if anyone has anything else to say

Reply:
I just merged in a bunch of patch updates. I'm going to try and update the minor updates before this goes in. I'll also bump this to 20. We typically don't have _too_ many deps for each of the repos, but combined it might add up. (i wish we could group it together)